### PR TITLE
Adding a game smash monty mole, contributing under GSSOC'24

### PR DESCRIPTION
**Pull request by rugved0102**
## PR Description 📜

Please include summary related to the issue you have fixed and describe your PR in brief over here by specifying the issue number on which you were working below
Fixes # <your_issue_number>

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in GameZone's README.md
- [x] I have added README.md in my Game folder 
- [ ] I have added working video of the game in README.md (optional)
- [ ] I have specified the respective issue number for which I have requested the new game.
- [ ] I have added my Game to gamesData.json file and it is loading properly on main website as intended.

<hr>

## Add your screenshots(Optional) 📸
image

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @rugved0102 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by ishita-43** [comment]
What is your issue number?

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Add your game entry into readme. Md file of the gamezone repo 
Add your game image into the assets folder of the gamezone

**Comment by rugved0102** [comment]
Done with the requested changes. Any suggested modification ??
I am contributing under gssoc'24 so I request you to label it

File: README.md [hunk 1]
```diff
@@ -1678,6 +1678,7 @@ This repository also provides one such platforms where contributers come over an
 |[Garden-Builder-game](https://github.com/kunjgit/GameZone/tree/main/Games/Garden-Builder-game)|
 
 | [Airhockey_game](https://github.com/kunjgit/GameZone/tree/main/Games/Airhockey_game)|
+|[Smash-Monty-Mole]()|
```
Comment (ishita-43) [inline README.md:1681]: Add your link also would be same as other only add your game folder name in the end

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please make following changes

**Comment by rugved0102** [comment]
done !

**Review by ishita-43 — APPROVED** [review]
Approved

**Comment by kunjgit** [comment]
Thank you  @rugved0102 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**